app/src/playStore/java/org/fossasia/openevent/general/search/SmartAuthViewModel.kt

```diff
@@ -23,9 +23,9 @@ const val RC_CREDENTIALS_SAVE = 3
 
 class SmartAuthViewModel : ViewModel() {
     private var requestedCredentialsEarlier = false
-    private lateinit var credentialsClient: CredentialsClient
-    private lateinit var signInClient: GoogleSignInClient
-    private lateinit var thisActivity: Activity
+    private var credentialsClient: CredentialsClient? = null
+    private var signInClient: GoogleSignInClient? = null
+    private var thisActivity: Activity? = null
```

Comment:
This is a very very poor implementation of ViewModel. I don't know what was I smoking when I let this get merged.

There are Android classes used in ViewModel. No doubt there was a crash :man_facepalming: 

Remove this feature completely and create an issue to reimplement this

Don't delete this ViewModel

Reply:
removed

Reply:
Revert this

Reply:
Done